bin/git-summary

```diff
@@ -211,14 +215,15 @@ print_summary_by_line() {
 
 print_summary() {
   if [ "$OUTPUT_STYLE" == "tabular" ]; then
-    tabular_headers="# Repo $SP Age $SP Last active $SP Active on $SP Commits $SP Uncommitted"
-    echo -e "$tabular_headers\n$project $SP $(repository_age) $SP $(last_active) $SP $(active_days $commit) days $SP $(commit_count $commit) $SP $(uncommitted_changes_count)" | column -t -s "$COLUMN_CMD_DELIMTER"
+    tabular_headers="# Repo $SP Age $SP Last active $SP Active on $SP Commits $SP Uncommitted $SP Branch"
+    echo -e "$tabular_headers\n$project $SP $(repository_age) $SP $(last_active) $SP $(active_days $commit) days $SP $(commit_count $commit) $SP $(uncommitted_changes_count) $SP $(current_branch_name)" | column -t -s "$COLUMN_CMD_DELIMTER"
   elif [ "$OUTPUT_STYLE" == "oneline" ]; then
-    echo "$project / age: $(repository_age) / last active: $(last_active) / active on $(active_days $commit) days / commits: $(commit_count $commit) / uncommitted: $(uncommitted_changes_count)"
+    echo "$project / age: $(repository_age) / last active: $(last_active) / active on $(active_days $commit) days / commits: $(commit_count $commit) / uncommitted: $(uncommitted_changes_count) / branch: $(current_branch_name)"
```

Comment:
Look like we need to escape the branch name, as it can contain `/`

Reply:
Do you mean by "escape" to just put it withing double quotes?

On the other hand, is it a problem if the branch name contains a `/`, because it is anyway just some output...

Reply:
I wonder if we escape the `/`, can awk or column handle it correctly?

Reply:
I guess if you want to `awk` this line, you always can use ` / ` (space before and after) to distinguish it from slashes in a text

Reply:
Maybe we can mention the `space before and after` in the doc?

Reply:
Done. I mentioned the separator for the tabular and the oneline option in the documentation and also that they are surrounded by spaces.